src/components/Legend.js

```diff
@@ -104,7 +164,28 @@ class Legend extends Component {
                   </div>
                 );
               })
-              .reverse()
+              .reverse()}
+          </div>
+        ) : null}
+
+        {drawVectorIcons ? (
```

Comment:
Moet nog een check in of er asset icons zijn op deze kaart (tile.assetTypes && tile.assetTypes.length)